Svc/FatalHandler/FatalHandlerComponentLinuxImpl.cpp

```diff
@@ -27,9 +27,9 @@ namespace Svc {
             FwEventIdType Id) {
         // for **nix, delay then exit with error code
         Fw::Logger::logMsg("FATAL %d handled.\n",(U32)Id,0,0,0,0,0);
+        (void)raise( SIGTERM );
```

Comment:
Why was this raise moved before the delay?

Reply:
It was done so that if some deployment contained a SIGTERM handler that does some cleanup (similarly to [Ref](https://github.com/nasa/fprime/blob/0e731d14c161a5c5dc37a2ce2e55009353806247/Ref/Top/Main.cpp#L17) it would allow for a cleaner exit. In case of SIGSEGV it doesn't make a difference.